R/tmb.R

```diff
@@ -262,8 +262,12 @@ h_mmrm_tmb_data <- function(formula_parts,
   }
   data <- data[data_order, ]
   weights <- weights[data_order]
-  data$weights <- weights
-  full_frame <- droplevels(stats::model.frame(formula_parts$full_formula, data = data, weights = weights))
+  data <- data.frame(data, weights)
+  # weights is always the last column
```

Comment:
nice! I would still add a test to show that this does not meddle an existing `weights` column in the `data`

Reply:
added now